components/suggestion/suggestion_box.jsx

```diff
@@ -302,8 +310,11 @@ export default class SuggestionBox extends React.Component {
                 provider.handleCompleteWord(term, matchedPretext);
             }
         }
+
+        // override if user finished typing term before results returned from API
         if (shouldEmitWordSuggestion) {
-            GlobalActions.emitCompleteWordSuggestion(this.suggestionId);
+            const override = pretext.endsWith(term);
```

Comment:
@hmhealey yeh, almost always this is false.
`term: miguel // my username handle`
`pretext: hey @mi // notice the handle is incomplete`
most all pretext follow this same format, this function is called on [ENTER].  Many times a user has an unfinished user handle that the suggeestion_box finishes for you.


The logic here is for overriding the completePending logic since it's not necessary. 
What ends up happening is a small hiccup in the typing experience, when usually you would need 2 [ENTER]s to send, you have to type 3 because it's in this completePending state (first ENTER)